### PR TITLE
using card component in recipe teaser template

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: web/themes/custom/pineapple/components/_card.twig [hunk 1]
```diff
@@ -2,13 +2,29 @@
 {% set attributes = attributes|default(create_attribute()).addClass([
   'card-container',
 ]) %}
-<div {{ attributes }}>
+{% if card_type == 'vertical'%}
+  <div {{ attributes.addClass('vertical') }}>
+{% else %}
+  <div {{ attributes.addClass('horizontal') }}>
+{% endif %}
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/components/_card.twig]: You don't need to do this, if you add `card_type` into the classes array below card-container It'll achieve the same result without the if statements.

Comment (katlazarova) [reply web/themes/custom/pineapple/components/_card.twig]: done

File: web/themes/custom/pineapple/templates/content/node--recipe--teaser.html.twig [hunk 2]
```diff
@@ -70,45 +70,32 @@
...
+{% if content.field_r_teaser_image|field_value %}
+  {% set image_url = bamboo_render_image_style(content.field_r_teaser_image|field_target_entity.field_media_image.entity.uri.value, 'horizontal_teaser') %}
+  {% set image_alt_text = content.field_r_teaser_image|field_target_entity.field_media_image.alt %}
+{% else %}
+  {% set image_url = bamboo_render_image_style(content.field_r_hero_image|field_target_entity.field_media_image.entity.uri.value, 'horizontal_teaser') %}
+  {% set image_alt_text = content.field_r_hero_image|field_target_entity.field_media_image.alt %}
+{% endif %}
+{% if "Yes" in content.field_r_highlighted.0 %}
+  {% set tag = content.field_r_highlighted|field_label %}
+{% endif %}
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/templates/content/node--recipe--teaser.html.twig]: You don't need to write an entire if statement here, you can use a ternary operator on line 88 to achieve the same result like so `node.field_r_highlighted.value == "1" ? content.field_r_highlighted|field_label` note here that 1 represents "Yes", when doing comparative statements it's usually better use **node.field_name.value**

Comment (katlazarova) [reply web/themes/custom/pineapple/templates/content/node--recipe--teaser.html.twig]: okay, thank you. Done.

File: web/themes/custom/pineapple/components/_card.twig [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
 <div {{ attributes }}>
   <div class="card-image-container">
     <img src="{{ image_url }}" alt="{{ image_alt_text }}">
+    {% if tag %}
+      <div class="highlighted-label">
+        <p>{{ tag }}</p>
+      </div>
+    {% endif %}
   </div>
   <div class="card-content u-o-3">
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/components/_card.twig]: Reduce owl class spacing value by one to `u-o-2`

Comment (katlazarova) [reply web/themes/custom/pineapple/components/_card.twig]: done

**Pull request merged**